.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
Hah, is this going to work here? (I assume it's mostly prospective given there isn't really a formatter for Ruby yet anyway.)

Reply:
The Rubocop extension does add an autocorrect command to VSCode, but it doesn't automatically run it even when `formatOnSave` is `true`... So yes, this is purely prospective for now.